builtin/logical/transit/path_wrapping_key.go

```diff
@@ -0,0 +1,100 @@
+package transit
+
+import (
+	"context"
+	"crypto/x509"
+	"encoding/pem"
+	"fmt"
+	"io"
+	"path"
+	"strconv"
+
+	"github.com/hashicorp/vault/sdk/framework"
+	"github.com/hashicorp/vault/sdk/helper/keysutil"
+	"github.com/hashicorp/vault/sdk/logical"
+)
+
+const WrappingKeyName = "wrapping-key"
+
+func (b *backend) pathWrappingKey() *framework.Path {
+	return &framework.Path{
+		Pattern: "wrapping_key",
+		Callbacks: map[logical.Operation]framework.OperationFunc{
+			logical.ReadOperation: b.pathWrappingKeyRead,
+		},
+		HelpSynopsis:    pathWrappingKeyHelpSyn,
+		HelpDescription: pathWrappingKeyHelpDesc,
+	}
+}
+
+func (b *backend) pathWrappingKeyRead(ctx context.Context, req *logical.Request, _ *framework.FieldData) (*logical.Response, error) {
+	p, err := getWrappingKey(ctx, req.Storage)
```

Comment:
This could technically race, leading to two keys being generated and returned to two callers.  Other users of the keysutil system use a LockManager, whose GetPolicy gets the instantiated key or creates it if missing, covered by a lock.  You might consider having a LockManager instance for this, even though it'll only be managing the one key.

Reply:
Updated the storage path for the wrapping key to be `policy/import/wrapping-key` rather than `import/policy/wrapping-key` which allowed use of the existing LockManager instance on the Transit backend. Since the `policy/` prefix is already in seal wrap storage, the extra `import/` prefix was removed. As far as I'm aware this shouldn't cause any name collisions, but if I'm missing a case I'll be more than happy to revisit this.